precode.sh

```diff
@@ -6,38 +6,57 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p ./task/dir3/dir4
```

Comment:
Можно в одну команду делать
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```